packages/node-api/src/interfaces/augment-api-events.ts

```diff
@@ -392,5 +404,15 @@ declare module '@polkadot/api-base/types/events' {
        **/
       [key: string]: AugmentedEvent<ApiType>;
     };
+    vote: {
+      /**
+       * Issued upon new Vote creation. [voteId, legalOfficers]
+       **/
+      VoteCreated: AugmentedEvent<ApiType, [u64, Vec<AccountId32>]>;
```

Comment:
So the list of voters is still part of the event after all? Currently not a big deal, but may become a problem in the future as the number of voters increases.

Reply:
pallet et node-api will be updated at the node runtime.
Reminder:
https://github.com/logion-network/logion-internal/issues/718